source/library/element-stub.js

```diff
@@ -0,0 +1,67 @@
+export default class ElementStub {
+	/**
+	 * Creates a new ElementStub instance. See {@link ElementStub#constructor}
+	 * @constructor
+	 * @private
+	 */
+	static create(...args) {
+		return new ElementStub(...args);
+	}
+
+	/**
+	 * Style proxy object
+	 * @type {Object}
+	 * @private
+	 */
+	style = {};
+
+	/**
+	 * Reference to currently active animationFrame
+	 * @type {Object}
+	 * @private
+	 */
+	frame = null;
+
+	/**
+	 * Creates a new ElementStub instance
+	 * @constructor
+	 * @param  {Node} element Node to stub
+	 * @param  {Window} [window=global.window] Global context to use
+	 * @return {ElementStub}         ElementStub instance
+	 * @private
+	 */
+	constructor(element, window = global.window) {
+		this.element = element;
+
+		const loop = () => {
+			this.render(this.element, this.style._style || this.style);
```

Comment:
`this.style` is the object `web-animations-js` writes to. Assignment to the `style` property is not needed because of the property initializer at [L17](https://github.com/marionebl/jogwheel/pull/157/files#diff-957c0874e7b325199edfaf2726d93bc5R16) :trollface: